src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_builders.py

```diff
@@ -623,25 +624,23 @@ def build(reader: LinesReader) -> Data:
                     )
                 )
 
-            well.calculated_document = CalculatedDocumentBuilder.build_quantity(
-                well_item
-            )
-
-        endogenous_control = get_str(results_metadata, "Endogenous Control")
-        reference_sample = get_str(results_metadata, "Reference Sample")
+            well.calculated_document = build_quantity(well_item)
```

Comment:
pre-existing bug that should nonetheless be fixed: if `well.items.values()` is empty, `well_item` have been assigned.